doc/libf3d/CLASSES.md

```diff
@@ -29,6 +29,10 @@ The window class is responsible for rendering the meshes. It supports multiple m
 
 * `EXTERNAL`: A window where the OpenGL context is not created but assumed to have been created externally. To be used with other frameworks like Qt or GLFW.
 
+* `EGL`: A window created offscreen, running with full graphical capabilities, possibly in a headless context.
+
+* `OSMESA`: A window created offscreen, running with software rendering, possibly in a headless context.
```

Comment:
maybe rework this part ? the all caps used to correspond to the enum but its only used internally now